README.md

```diff
@@ -21,7 +44,8 @@ Install [Bazelisk](https://github.com/bazelbuild/bazelisk) and [Buildifier](http
 
 ### Options
 
-You can specify a `bazelisk_version` and a `buildifier_version` to pin each of the tools like this:
+You can specify a `bazelisk_version` and a `buildifier_version` to pin each of
+the tools like this:
```

Comment:
As you are at it, can you please add the `v` before the version string (like in the default option) which I fixed in the tests earlier?